modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/operations/util/SyncmlCredentials.java

```diff
@@ -29,8 +30,7 @@
  */
 public class SyncmlCredentials {
 
-    public String generateCredData(String nextNonce) throws UnsupportedEncodingException,
-            NoSuchAlgorithmException {
+    public String generateCredData(String nextNonce) throws SyncmlMessageFormatException {
```

Comment:
do not see any instant variables in this Class definition. Thus, there is no need as it seems to instantiate this class. Better to make this a utility class and methods static.